crates/primitives-traits/src/lib.rs

```diff
@@ -86,6 +86,10 @@ pub mod receipt;
 pub use receipt::{FullReceipt, Receipt};
 
 pub mod transaction;
+pub use alloy_consensus::{
+    transaction::{PooledTransaction, Recovered, TransactionMeta},
+    ReceiptWithBloom, TxType,
+};
```

Comment:
cool, these are nice to have, but I'd like to not re-export the PooledTransaction and TxType because these are eth specific